index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Bom dia, Artur. Que tal mudar o atributo lang para pt-br?
```suggestion
<html lang="pt-br">
```